### PR TITLE
Integrate add interview command with gui

### LINKED ISSUE #92: Integrate Add Inteview Command with GUI
Gui is already refactored by @likeabowx to include tabs for interview and positions.

Need to update the logic component in order to reflect updated changes in the hireLah GUI

### PR DESCRIPTION
**Pull request by goalfix**
Main changes:
- Fix bug in add interview command pulling twice from applicant list instead of once from applicant and once from position
- add integration for interview command with GUI

Closes #92

### CODE REVIEW

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -67,6 +70,26 @@
         };
     }
 
+    public static Interview[] getInterviewPositions() {
+        Applicant[] samplePersons = getSamplePersons();
+        Position[] samplePositions = getSamplePositions();
```
Comment (likeabowx) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:75]: Why is this called `getInterviewPositions()` shouldn't it be `getSampleInterviews()` like the other methods?

**Review by likeabowx — CHANGES_REQUESTED** [review]
There seems to be a typo for the method name highlighted, looks good otherwise.

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 1 above)
Comment (goalfix) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java:75]: Thanks for pointing this out! It's indeed a typo. Will fix it now.

**Review by likeabowx — APPROVED** [review]
LGTM after fixing the typo.

**Pull request merged**